app/v2/models/MessageType.scala

```diff
@@ -25,6 +25,8 @@ import play.api.libs.json.Writes
 abstract class MessageType(val code: String, val rootNode: String) extends Product with Serializable
 
 object MessageType {
+  case object AmendmentAcceptance              extends MessageType("IE004", "CC004C")
```

Comment:
these also need to be added to the departures array below - which we populate the message type action with to ensure the message type sent is valid

Reply:
You're correct - I had to rebase this because of IE029 had already been done and lost it!